src/argus_ticket_rt.py

```diff
@@ -63,15 +63,17 @@ def get_custom_fields(
             serialized_incident["end_time"] = serialized_incident["end_time"][:-6]
         else:
             del serialized_incident["end_time"]
+
         incident_tags = RequestTrackerPlugin.convert_tags_to_dict(
             serialized_incident["tags"]
         )
-        custom_fields = ticket_information.get("custom_fields_set", {})
+        custom_fields = dict()
+        custom_fields.update(ticket_information.get("custom_fields_set", {}))
```

Comment:
You could also get in the habit of using copy.deepcopy:

`custom_fields = deepcopy(ticket_information.get("custom_fields_set", {}))`

or kwargs-magic (won't copy dicts in dicts):

`custom_fields = dict(**ticket_information.get("custom_fields_set", {}))`

Many ways to solve it, very classic python problem. The same can occur with any mutable collection type, like `list()`.

Reply:
Yes, I first solved it with deepcopy, but I think this solution looks nicer